datalad/distribution/uninstall.py

```diff
@@ -1,157 +0,0 @@
-# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
-# ex: set sts=4 ts=4 sw=4 et:
-# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
-#
-#   See COPYING file distributed along with the datalad package for the
-#   copyright and license terms.
-#
-# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
-"""Thin shim around drop to preserve some backward-compatibility
-
-"""
-
-__docformat__ = 'restructuredtext'
-
-import logging
-
-from datalad.support.param import Parameter
-from datalad.support.constraints import (
-    EnsureStr,
-    EnsureNone,
-)
-from datalad.distribution.dataset import (
-    datasetmethod,
-    require_dataset,
-    Dataset,
-    EnsureDataset,
-)
-from datalad.interface.base import Interface
-from datalad.interface.common_opts import (
-    if_dirty_opt,
-    recursion_flag,
-)
-from datalad.interface.utils import (
-    eval_results,
-    handle_dirty_dataset,
-)
-from datalad.interface.results import get_status_dict
-from datalad.interface.base import build_doc
-from datalad.utils import (
-    ensure_list,
-)
-from datalad.core.local.status import get_paths_by_ds
-
-
-lgr = logging.getLogger('datalad.distribution.uninstall')
-
-dataset_argument = Parameter(
-    args=("-d", "--dataset"),
-    metavar="DATASET",
-    doc="""specify the dataset to perform the operation on.
-    If no dataset is given, an attempt is made to identify a dataset
-    based on the `path` given""",
-    constraints=EnsureDataset() | EnsureNone())
-
-
-check_argument = Parameter(
-    args=("--nocheck",),
-    doc="""whether to perform checks to assure the configured minimum
-    number (remote) source for data.[CMD:  Give this
-    option to skip checks CMD]""",
-    action="store_false",
-    dest='check')
-
-
-@build_doc
-class Uninstall(Interface):
-    """DEPRECATED: use the `drop` command"""
```

Comment:
I wondered when was it deprecated... Had to consult changelog, seems in
0.16.0 (Fr Apr 8 2022), so not even a month ago . I wonder now if removing not too early?

Reply:
Ok, closing. Please let me know when a better time would be.